gateway/src/apicast/configuration_loader/remote_v2.lua

```diff
@@ -259,6 +243,83 @@ function _M:index(host)
   return nil, 'invalid status'
 end
 
+-- Returns existing proxy configs in a single page
+-- @param http_client the http client object
+-- @param portal_endpoint 3scale API endpoint
+-- @param host proxy config filter based on request hostname. Optional, can be nil.
+-- @param env gateway environment
+-- @param page page in the paginated list. Defaults to 1 for the API, as the client will not send the page param.
+-- @param per_page number of results per page. Default and max is 500 for the API, as the client will not send the per_page param.
+local proxy_configs_per_page = function(http_client, portal_endpoint, host, env, page, per_page)
```

Comment:
Why is it simply not a function? e.g. `function proxy_configs_per_page(..)...`

Reply:
fixed